### PR TITLE
Dialog remove 80

### PR DESCRIPTION
**Pull request by robinkwilson**
DialogExternal on an app server uses port 8443 (ret is using 443 on app server)
DialogExternal on a streaming server uses port 443
Turn is always on 5349

### CODE REVIEW

File: cloudformation/stack.yaml [hunk 1]
```diff
@@ -3797,7 +3798,7 @@ Outputs:
 
   JanusExternalPort:
     Description: Janus External Port [reticulum/janus/janus_port]
-    Value: !FindInMap [ServicesMeta, DialogExternal, Port]
+    Value: !If [HasStreamingServers, !FindInMap [ServicesMeta, DialogExternal, StreamPort], !FindInMap [ServicesMeta, DialogExternal, AppPort] ]
```
Comment (brianpeiris) [inline cloudformation/stack.yaml:3801]: I noticed that `HasStreamingServers` is false when the stack is set to Offline. I don't think it will break anything, but it's a bit surprising, so perhaps another thing to keep in mind if we run into any quirks later.

**Review by brianpeiris — APPROVED** [review]
LGTM. Just commented some thoughts.

File: cloudformation/stack.yaml [hunk 2]
```diff
@@ -370,9 +370,10 @@ Mappings:
     DialogInternal:
       Port: 8443
     DialogExternal:
-      Port: 80
+      AppPort: 8443
+      StreamPort: 443
     DialogTurnExternal:
-      Port: 443
+      Port: 5349
```
[the remark this answers is not in the document]
Comment (robinkwilson) [reply cloudformation/stack.yaml:376]: Turn on a personal 1 server is using 5349 anyway IIRC. I also agree. We'll see after this is put in practice.

I'd love to get this configurable. But definitely in a future template update.

File: cloudformation/stack.yaml (hunk 1 above)
Comment (robinkwilson) [reply cloudformation/stack.yaml:3801]: I think that's correct. There's neither streaming nor app servers when Offline mode is on since there's no EC2 instances up.

**Comment by robinkwilson** [comment]
## QA Test Plan

### TLDR PR context
This template change updates the ports used for connecting to Hubs Cloud deployments and removes our dependence on port 80 (commonly blocked by enterprise networks). When port 80 is blocked on the current Hubs Cloud deployments, users get stuck on the loading screen and will never enter the room or hear people.

**_I will combine all of these template changes into one for a thorough hubs test._**

At a minimum for testing just this PR change:
We need to verify that with port 80 blocked by a firewall, you can enter a room using Turn, or foregoing Turn and still have sound, can talk to people.

## Plan

I will provide template links via email.

### Update existing stack with a new template (users with existing stacks will follow this workflow)
#### For Personal and Enterprise multi-server
1. AWS Cloudformation
2. Click on your stack
3. Click "Update"
4. Select "Replace current template"
5. Paste in new template URL
6. Click "Next"
7. No need to change any stack parameters so scroll through and finish the update
8. Deploy then follow "Verify below cases work for each deployment"

### New stacks, create a new stack with template
#### For Personal and Enterprise multi-server
1. AWS Cloudformation
2. Click "Create stack"
3. Paste in new template URL
4. Fill in params as you would create a new stack :)
5. Deploy then follow "Verify below cases work for each deployment"

### Verify below cases work for each deployment
1. Run stack
2. Create room
3. Join room with 2 or more people
   1. Use query params `?force_tcp=true`, `?force_turn=true`, as well as `?force_turn=true&force_tcp=true`
   2. Use no query params
   3. Try blocking port 80 tcp and udp (I used windows firewall to block 80 for my tests)
4. Ensure every person can enter the room, hear everyone, move around, bring in images and assets ("the usual hubs stuff")
5. Ensure you can login
6. Setup ssh and ensure you can ssh into the box (make sure we didn't mess up port 22)
   1. In the admin panel add 2FA in Server Access
   2. SSH into box
7. Success, awesome! Done verified :D 

THANK YOU!! You're awesome! Happy to answer any questions around this :)

**Comment by Utopiah** [comment]
Unfortunately such changes didn't seem to work in my restrictive context. JanusExternal.Port to 8443 and JanusTurnExternal.Port to 443 did work though, but not the other way around.
